tests/src/test/fdc3.broadcast.ts

```diff
@@ -1,9 +1,358 @@
+import { ResolveError, Listener } from "@finos/fdc3";
+import { assert, expect } from "chai";
+
 export default () =>
   describe("fdc3.broadcast", () => {
+    let listener: Listener;
+    let listener2: Listener;
+
+    let channelsAppContext = {
+      type: "channelsAppContext",
+      reverseFunctionCallOrder: false,
+      contextBroadcasts: {
+        instrument: true,
+        contact: false,
+      },
+    };
+
+    afterEach(async () => {
+      if (listener !== undefined) {
+        await listener.unsubscribe();
+        listener = undefined;
+      }
+
+      if (listener2 !== undefined) {
+        await listener2.unsubscribe();
+        listener2 = undefined;
+      }
+
+      await window.fdc3.leaveCurrentChannel();
+      channelsAppContext.reverseFunctionCallOrder = false;
+      channelsAppContext.contextBroadcasts.contact = false;
+    });
+
     it("Method is callable", async () => {
       await window.fdc3.broadcast({
         type: "fdc3.instrument",
         id: { ticker: "AAPL" },
       });
     });
+
+    it("App A adds context listener then joins channel 1 => App B joins channel 1 then broadcasts context => App A receives context from B", async () => {
+      const asyncWrapper = () => {
```

Comment:
Just wondering what the purpose of the asyncWrapper is?

The out function is async, i.e. `it("foo", async () => { //async stuff here })`, so am not sure why it is needed?

Reply:
You are right. I've removed the async wrapper